### ISSUE TITLE
problem: side and top navbar layout is the same for small and large screens

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#631]
Problem: in light of good UX design rules (http://uxmovement.com/navigation/top-navigation-vs-left-navigation-which-works-better/), we should have critical menu items in the top navbar when it's possible (e.g. on screens with a resolution wider than x) and these should be moved to the side menu on smaller screens. E.g. Bounties should probably be top-left right next to the side menu button.

This isn't possible at the moment because there's no *framework* to to handle these changes to the nav bar and side menu. I guess having a small screen and large screen layout is enough, not sure what width we should use but this is easy to change anyway. 

Once the framework is in place we can easily change which items are displayed on each menu. Please document the code well with any solution so that it's clear how to do this.

**Comment by MadanBhandari** [comment]
I was also looking about the navigation for the website, I am thinking about having just single navigation should be enough, no more confusion, more space for content.

Here are few points, I would like to share about navigation:

- Use one navigation, in this case top navigation, on smaller screen it will become collapsible
- Combine menu items into groups for eg. Wallet, Pending Items, Edit Profile can go into single category (which are user specific and only to be shown after login) - inside the user name
- We don't need separate edit and my profile on navigation, edit profile can go inside my profile
- Can we combine activity and wallet notifications into single notification ? Or this should be separate as it is now.

ping @gazhayes

**Comment by gazhayes** [comment]
I think it's worth having both top and side navigation as we currently do. 

There are huge benefits to a top menu because the user sees the top-left item immediately. It shouldn't be used for too many items, probably 3 at the most (bounties, add currency, and maybe compare currencies). Site-wide search could be incorporated there later too. What we put here immediately tells users what the site is all about and gets them involved.

At the same time, a side menu is really good for having a large number of items because users can very quickly scan a vertical menu, it's much faster then hunting through sub-sections on a horizontal navigation.

Having sub-menus is annoying, it's necessary sometimes but it's also evil. They should only be used when they *really* make intuitive sense (and *not* to 'remove clutter), otherwise people get annoyed trying to find things.

> Can we combine activity and wallet notifications into single notification
No, because wallet will become more featured later (withdraw, deposit, trade, etc). Wallet and activity notifications need some serious UX work though but this is a separate issue.

**Comment by gazhayes** [comment]
Also the side menu works very well on mobile, I think it's much better than using those top menus that convert to a vertical menu and take up 100% of the screen even when they don't need to. We probably need to make sure it can be scrolled in case it gets cut off on some screens, but apart from that I think it's pretty good already.

(I'm not saying the design itself is good, just the functionality).

**Comment by samuelralak** [comment]
@gazhayes I believe space can still be created on the top nav for smaller devices. kindly to this comment [here](https://github.com/Blockrazor/blockrazor/issues/727#issuecomment-375916584). We should also identify the critical menu items that need to go to the top nav for smaller screens.

**Comment by samuelralak** [comment]
@gazhayes what the update on this one?

**Comment by gazhayes** [comment]
No update, just what you see here.

**Comment by AshHimself** [comment]
Fixed in #1082

**Comment by AshHimself** [comment]
@blockrazorbot done

**Comment by gazhayes** [comment]
How do you add large screen vs small screen items? Can you add some comments in the code?

**Comment by AshHimself** [comment]
I can but its all document here https://getbootstrap.com/docs/4.0/utilities/display/

**Comment by gazhayes** [comment]
Oh I know, and it's pretty easy for us, but you should make more comments anyway (on this and in general) so that it's *really* clear what's going on to people who are learning for the first time.

**Comment by AshHimself** [comment]
Roger that

**Comment by AshHimself** [comment]
@gazhayes - I added some html comments in my last PR. Did you have another idea in mind to document it? README?

**Comment by gazhayes** [comment]
Nah that's good.
